projects/canopy/src/lib/hero-img/hero-img.notes.ts

```diff
@@ -0,0 +1,88 @@
+export const notes = (imageBackgroundHeroHTML: string) => `
+  # Hero Image Component
+  
+  
+  ## Purpose
+  Provides a hero component that can used to display a hero banner with a background image and a content container. Content container can be used to display page title, subtitle and buttons.
+  
+  ## Usage
+  ~~~js
+  @NgModule({
+    ...
+    imports: [ ..., LgHeroImgModule ],
+  })
+  ~~~
+  
+  and in your HTML...
+  
+  ~~~html
+  ${imageBackgroundHeroHTML}
+  ~~~
+  
+
+  #### Inputs
+  
+  | Name | Description | Type | Default | Required |
+  |------|-------------|:----:|:-----:|:-----:|
+  | \`\`imageUrl\`\` | Background image url | string | n/a | Yes |
+  
+  
+  ### LgHeroImgComponent
+  This is the hero banner with a background image that runs across
+  the top of the page. It also controls the functionality to create the 'overlap' between the page content.
+
+  #### Configure Overlap
+
+  ~~~html
+  <lg-hero-img [overlap]="2"></lg-hero-img>
+  ~~~
+
+  #### Inputs
+
+  | Name | Description | Type | Default | Required |
+  |------|-------------|:----:|:-----:|:-----:|
+  | \`\`overlap\`\` | The amount that the page content overlaps the hero component (rem) | number | 2 | No |
+    
+  ### LgHeroImgContentComponent
```

Comment:
I'm wondering we need this `LgHeroImgContentComponent` component? the background image seem to be already in the `LgHeroImgComponent`?